### DISCUSSION TITLE
Information on #1340

### DISCUSSION THREAD

**Discussion by testinggithubuser490** [bromite/bromite#1498] [General]
Just wondering why it was closed? I am still very much having these issues on the past three builds now. I've switched to Kiwi for the time being and no issues there.

What's odd is for me, at first login the page and feed loads fine. However when I do a pull to refresh, then it only grabs one single post, and just stays suck on the (getting posts) animation. As a workaround, I can load the page in desktop mode and that seems to appear to grab all of the assets. When I switch back to mobile then I can interact just fine until it runs out of cached assets, or I try to check my notifications.

**Comment by csagan5** [comment]
> Just wondering why it was closed?

The version is not valid.

> I've switched to Kiwi for the time being and no issues there.

Please read the [pinned post](https://github.com/bromite/bromite/discussions/893) before posting; this is not OK to write here.

**Reply by testinggithubuser490** [reply]
> 
> 
> > Just wondering why it was closed?
> 
> The version is not valid.
> 
> > I've switched to Kiwi for the time being and no issues there.
> 
> Please read the [pinned post](https://github.com/bromite/bromite/discussions/893) before posting; this is not OK to write here.

Thank you for letting me know, sorry if I offended. Should clarify I switched to Kiwi solely for Facebook usage, I still have Bromite installed and use it for every other site, I really appreciate your work. That's on me for wording things poorly.
